Api.Flickr/Models/FlickrPhotosetsGetPhotosResponseStatus.cs

```diff
@@ -0,0 +1,15 @@
+namespace Api.Flickr.Models
+{
+    /// <summary>
+    /// Statuses of response for photosets.getPhotos method
+    /// </summary>
+    public enum FlickrPhotosetsGetPhotosResponseStatus
+    {
+        Okay,
+        PhotosetNotFound,
+        UserNotFound,
```

Comment:
401

Reply:
We don't use any credentials for making requests, so it's redundand